sDownloads.ps1

```diff
@@ -0,0 +1,27 @@
+# Check directory sDownloads.ps1 runs from for sd-config file
+$ConfigPath = $PSScriptRoot + "\sd-config"
+
+# sd-config needs to stay with sDownloads.ps1
+# Change the path if you want sd-config somewhere else
```

Comment:
Since the config path always needs to be known, could you pass that path as an option when running the script? You can still have a default if nothing it passed.